SparkleSpin/PlayerViewModel.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  PlayerViewModel.swift
+//  SparkleSpin
+//
+//  Created by Britney Smith on 2/27/19.
+//  Copyright © 2019 Britney Smith. All rights reserved.
+//
+
+import Foundation
+
+class PlayerViewModel {
+    
+    private let playerModel: PlayerModel
+    var nameString: String? = nil
+    
+    init(playerModel: PlayerModel) {
+        self.playerModel = playerModel
+    }
+    
+    func updateProperties(playerModel: PlayerModel, nameString: String) {
+        setPlayerName(playerModel: playerModel, nameString: nameString)
+    }
+    
+    private func setPlayerName(playerModel: PlayerModel, nameString: String?) {
+        playerModel.name = nameString
+    }
```

Comment:
What's the purpose of having this private function? I think you could call directly `playerModel.name = nameString` in `updateProperties`  but maybe I'm missing something.

Reply:
I think I did this in anticipation of `PlayerModel` soon having a property of type `Chore`, but there could be a better way of handling that when I get to it. For now, I will call `playerModel.name = nameString` directly.